project-base/app/src/Component/FileUpload/FileUpload.php

```diff
@@ -79,11 +78,15 @@ private function getPositionForNewEntity(EntityFileUploadInterface $entity): int
         $entityId = $entity->getEntityId();
         $type = $entity->getType();
         $uploadEntityType = $this->getUploadEntityType($entity);
+        $key = sprintf('%s~%s~%s~%s', $entityName, $entityId, $type, $uploadEntityType);
+
+        if ($this->localCacheFacade->hasItem(self::POSITION_BY_ENTITY_AND_TYPE_CACHE_NAMESPACE, $key)) {
```

Comment:
what about `getOrSave()`? :innocent:

Reply:
in this case is not possible i think, so we can discuss personally.